components/diagnosis-suggestion/api/pom.xml

```diff
@@ -29,7 +29,7 @@
   <name>PhenoTips - Diagnosis Suggestion Java Code</name>
 
   <properties>
-    <coverage.instructionRatio>0.81</coverage.instructionRatio>
+    <coverage.instructionRatio>0.78</coverage.instructionRatio>
```

Comment:
Since this commit fixes a bug, you should add a test that checks the fix stays in place, so that a future regression will be caught by the test. That's why some tests are called "regression tests".